core/renderers/block_rendering_rewrite/block_rendering_constants.js

```diff
@@ -19,125 +19,167 @@
-Blockly.blockRendering.constants.BETWEEN_STATEMENT_PADDING_Y = 4;
+  // This is the max width of a bottom row that follows a statement input and
+  // has inputs inline.
+  this.MAX_BOTTOM_WIDTH = 66.5;
 
-// This is the max width of a bottom row that follows a statement input and
-// has inputs inline.
-Blockly.blockRendering.constants.MAX_BOTTOM_WIDTH = 66.5;
+  /**
+   * Height of the top hat.
+   * @const
+   * @private
+   */
+  this.START_HAT_HEIGHT = 15;
 
-/**
- * Height of the top hat.
- * @const
- * @private
- */
-Blockly.blockRendering.constants.START_HAT_HEIGHT = 15;
+  /**
+   * Width of the top hat.
+   * @const
+   * @private
+   */
+  this.START_HAT_WIDTH = 100;
 
-/**
- * Width of the top hat.
- * @const
- * @private
- */
-Blockly.blockRendering.constants.START_HAT_WIDTH = 100;
+  this.SPACER_DEFAULT_HEIGHT = 15;
 
-Blockly.blockRendering.constants.SPACER_DEFAULT_HEIGHT = 15;
+  this.MIN_BLOCK_HEIGHT = 24;
 
-Blockly.blockRendering.constants.MIN_BLOCK_HEIGHT = 24;
+  this.EMPTY_INLINE_INPUT_PADDING = 14.5;
 
-Blockly.blockRendering.constants.EMPTY_INLINE_INPUT_PADDING = 14.5;
+  this.EMPTY_INLINE_INPUT_HEIGHT = this.TAB_HEIGHT + 11;
 
-Blockly.blockRendering.constants.EMPTY_INLINE_INPUT_HEIGHT =
-    Blockly.blockRendering.constants.TAB_HEIGHT + 11;
+  this.EXTERNAL_VALUE_INPUT_PADDING = 2;
 
-Blockly.blockRendering.constants.EXTERNAL_VALUE_INPUT_PADDING = 2;
+  /**
+   * The height of an empty statement input.  Note that in the old rendering this
+   * varies slightly depending on whether the block has external or inline inputs.
+   * In the new rendering this is consistent.  It seems unlikely that the old
+   * behaviour was intentional.
+   * @const
+   * @type {number}
+   */
+  this.EMPTY_STATEMENT_INPUT_HEIGHT = this.MIN_BLOCK_HEIGHT;
 
-/**
- * The height of an empty statement input.  Note that in the old rendering this
- * varies slightly depending on whether the block has external or inline inputs.
- * In the new rendering this is consistent.  It seems unlikely that the old
- * behaviour was intentional.
- * @const
- * @type {number}
- */
-Blockly.blockRendering.constants.EMPTY_STATEMENT_INPUT_HEIGHT =
-    Blockly.blockRendering.constants.MIN_BLOCK_HEIGHT;
+  this.START_POINT = Blockly.utils.svgPaths.moveBy(0, 0);
 
-Blockly.blockRendering.constants.START_POINT = Blockly.utils.svgPaths.moveBy(0, 0);
+  /**
+   * Height of SVG path for jagged teeth at the end of collapsed blocks.
+   * @const
+   */
+  this.JAGGED_TEETH_HEIGHT = 12;
 
-/**
- * Height of SVG path for jagged teeth at the end of collapsed blocks.
- * @const
- */
-Blockly.blockRendering.constants.JAGGED_TEETH_HEIGHT = 12;
-/**
- * Width of SVG path for jagged teeth at the end of collapsed blocks.
- * @const
- */
-Blockly.blockRendering.constants.JAGGED_TEETH_WIDTH = 6;
+  /**
+   * Width of SVG path for jagged teeth at the end of collapsed blocks.
+   * @const
+   */
+  this.JAGGED_TEETH_WIDTH = 6;
```

Comment:
As discussed in person, I think everything that's a fixed constant (up to this point) should also be pulled out into one overridable method, but that's something to play with in a separate PR.